backend/audit/logXML.py

```diff
@@ -20,7 +20,7 @@
 from xml.etree import ElementTree, cElementTree
 from xml.dom import minidom
 
-auditNum = "1"
+auditNum = "45"
```

Comment:
Can we add some kind of command line statement so we don't have to go into this file and change it every time? Not a big concern for this pull request, just some super minor thing to think about.

Reply:
Yup! I'll try to add one real quick.